src/Postgre/Connection.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace atk4\dsql\PgSQL;
+namespace atk4\dsql\Postgre;
```

Comment:
I just written my friend working with this database daily. He told me that only two names are used for this DB - Postgres and Postgresql. I also asked him about PgSQL -> Pgsql, but he told me, that they almost never use this name for it.

Postgres is an old/alias name for Postgresql based on Wiki. So I would you full "Postgresql" here. Wdyt?

Reply:
Aim is to be as intuitive and standard as possible and build the magic on top of that so `Postgresql` is fine with me.